src/QsCompiler/TestProjects/test18/Bell.qs

```diff
@@ -1,15 +1,5 @@
-﻿// TODO: Uncomment this once references are fixed for notebooks
```

Comment:
Removing this since I don't think we'll have the reference dlls on-disk when the unit tests run